src/state/user/actions.ts

```diff
@@ -58,3 +63,4 @@ export const addWatchlistPool = createAction<{ address: string }>('user/addWatch
 export const hidePhishingWarningBanner = createAction<void>('user/hidePhishingWarningBanner')
 
 export const setIsExchangeChartDisplayed = createAction<boolean>('user/toggleIsExchangeChartDisplayed')
+export const setChartViewMode = createAction<ChartViewMode>('user/toggleChartViewMode')
```

Comment:
super nitpick - the action name should be `user/setChartViewMode` 😊

Reply:
```suggestion
export const setChartViewMode = createAction<ChartViewMode>('user/setChartViewMode')
```